lib/OpenQA/Schema/ResultSet/Jobs.pm

```diff
@@ -317,9 +317,16 @@ sub prepare_complex_query_search_args ($self, $args) {
             push(@conds, {'me.id' => {-in => $subquery->get_column('job_id')->as_query}});
         }
 
-        for my $key (qw(build distri version flavor arch test machine)) {
+        for my $key (qw(distri version flavor arch test machine)) {
             push(@conds, {"me." . uc($key) => $args->{$key}}) if $args->{$key};
         }
+        if (defined $args->{build}) {
+            my $build = $args->{build};
+            if (ref $build eq 'ARRAY') {
+                push(@conds, {"me.BUILD" => {-in => $build}});
+            }
+            else { push(@conds, {"me.BUILD" => $build}) }
+        }
```

Comment:
```suggestion
        if (my $build = $args->{build}) {
                push @conds, ref $build eq 'ARRAY' ? {"me.BUILD" => {-in => $build}} : {"me.BUILD" => $build};
        }
```

Reply:
I *love* the ternary operator and it's something I look for in all programming languages I use :)